docs/docs/spark-configuration.md

```diff
@@ -190,6 +190,7 @@ df.write
 | compression-codec      | Table write.(fileformat).compression-codec | Overrides this table's compression codec for this write      |
 | compression-level      | Table write.(fileformat).compression-level | Overrides this table's compression level for Parquet and Avro tables for this write |
 | compression-strategy   | Table write.orc.compression-strategy       | Overrides this table's compression strategy for ORC tables for this write |
+| distribution-mode | `Range` if Sort Order Defined ; `Hash` if Partition Defined ; `None` otherwise | Override this table's distribution mode for this write |
```

Comment:
Can we change this to a link to the spark-writes section? May be more clear

Reply:
@RussellSpitzer good point, I just linked from both sections, let me know what you think now?